### DISCUSSION TITLE
Add Wattage Sliders For GPU

### DISCUSSION THREAD

**Discussion by TheSpankMan** [seerge/g-helper#2780] [Ideas]
Hello seerge, love the G-Helper application.

Could you add GPU wattage sliders under the GPU tab? Although the temperature targets slider serves a similar function, I find setting the wattage to a specific number could potentially provide more stable performance and frame rates especially if temperatures stay under the target.

Thanks.

**Comment by seerge** [comment]
@TheSpankMan hello, if your device supports adjustable GPU power - you will see that already. If you don't see them - it simply means it's not possible on your model.

Only 2024 models support custom GPU power

Thanks.

**Reply by TheSpankMan** [reply]
@seerge, I see! I'm currently running a 2023 Duo and only see the Dynamic Boost slider so that is unfortunate that custom GPU power isn't supported.

Thanks for the quick response!

**Reply by seerge** [reply]
You can limit max GPU clocks still, that also (indirectly) limits power.
